index.html

```diff
@@ -808,6 +811,90 @@ <h3>
           </p>
         </section>
       </section>
+      <section>
+        <h3>
+          `id` member
+        </h3>
+        <p>
+          The [=manifest's=] <code><dfn data-export="" data-dfn-for=
+          "manifest" id="identity">id</dfn></code> member is a <a>string</a> that
+          represents the <dfn>identity</dfn> for the application. 
```

Comment:
why is it scoped to web-application that's not used for any other fields?

Reply:
I guess it's ok to leave it unstopped for now... I have a feeling it will eventually get used in other (external) contexts, which is why I suggested it. However, we can deal with that when it actually happens.